components/form-builder/util.ts

```diff
@@ -251,3 +251,11 @@ export const allowedTemplates = [
   FormElementTypes.firstMiddleLastName,
   FormElementTypes.contact,
 ] as const;
+
+export const isVaultDelivery = (deliveryOption?: { emailAddress?: string } | null) => {
```

Comment:
deliveryOption should not be an optional parameter --- without it your not doing any sort of check.

Reply:
good catch!

Reply:
Changed it to 
```javascript
export const isVaultDelivery = (deliveryOption: DeliveryOption | undefined)
```

let me know if that's better or not @timarney